### ISSUE TITLE
Playwright - Editor Transition: Post Lifecycle 

### ISSUE THREAD

**Issue by dpasque** [Automattic/wp-calypso#55950]
### Details

See overall transition details here: pciE2j-sq-p2

Coverage:
- Focus is on walking a single post through all the statuses it can have
- Each status is seen and interacted with through Calypso post page
- Publishing, editing, reverting to draft, scheduling, trashing, hard deleting
** no need to do for both page and post, so picking post here
** will use basic content with no images, just a title & paragraph

Includes coverage from prior specs:
- wp-calypso-gutenberg-post-editor-spec.js

**Comment by worldomonation** [comment]
I'm going to split the spec file `wp-calypso-gutenberg-post-editor-spec.js` into multiple smaller specs, each standing on its own. 

The way `wp-calypso-gutenberg-post-editor-spec.js` is written makes it difficult to:
- isolate the feature under test
- follow along the history of the spec file for a given feature

as well as making the resulting spec file overly long.

**Comment by dpasque** [comment]
@worldomonation

>I'm going to split the spec file wp-calypso-gutenberg-post-editor-spec.js into multiple smaller specs, each standing on its own.

As a heads up -- this is what I already did in my initial test plan mapping! This issue wording may be kind of confusing, but this new spec is only intended to have _pieces_ of that original post editor spec. 

Feel free to redesign my test plan map if you want! I just wanted to give you a heads up that I already did that splitting to potentially help save you some work! 

Details of that test plan map can be found here: pciE2j-sq-p2
